### DISCUSSION TITLE
Duplicate Page content seen with different metadata as a result of space text splitter in linux

### DISCUSSION THREAD

**Discussion by swathithiyan** [explosion/spaCy#12959] [Help: Other Questions]
Hi

We are using spacy text_splitter to chunk a document and write into chroma DB.
What we see that , post chunking(docs = text_splitter.split_documents(documents)), we are seeing duplicate page contents with different metadata, refer the screenshot, key is the duplicate page content , value is the number of times , its repeated and list of metadata.
<img width="1097" alt="Screenshot 2023-09-05 at 9 38 45 PM" src="https://github.com/explosion/spaCy/assets/125479417/0db8ced1-c2c2-43c8-830a-5e11661e6b36">

what's strange is that , issue is happening in linux OS, in MAC OS it works perfectly. 
spacy version used is 3.5.1.
can you pls help here.

**Comment by svlandeg** [comment]
Hi!

When creating a new issue or discussion topic, it would be beneficial to paste the code you ran (a minimal reproducable example) and to paste examples instead of taking screenshots, which are difficult for us to process.

That said, I assume you're using [Langchain's SpacyTextSplitter](https://api.python.langchain.com/en/latest/text_splitter/langchain.text_splitter.SpacyTextSplitter.html) which has not been created by us.
